src/factory_sim/objectives/pick_up_tool_from_holder.xml

```diff
@@ -39,8 +45,8 @@
       <Action
         ID="SetupMTCMoveAlongFrameAxis"
         hand_frame="tool0"
-        max_distance="0.095"
-        min_distance="0.095"
+        max_distance="0.1"
```

Comment:
During development, I found I needed the end effector to get a little closer to the tool to get it to activate.